beetmoverscript/script.py

```diff
@@ -313,6 +316,16 @@ def list_bucket_objects(context, s3_resource, prefix):
                     upload_to_s3(context=context, s3_key=destination, path=source)
                 )
             )
+
+            if is_partner_public_task(context):
+                # we trim the full destination to the part after
+                # candidates/61.0b2-candidates/buildN/
+                artifact_pretty_name = destination[destination.find(locale):]
```

Comment:
Nit, not blocking: I wonder if it makes sense to guard this block code with something like https://github.com/mozilla-releng/beetmoverscript/pull/150/files#diff-98ba072d44cc67fadf38dc459a83d934R287 in order to make sure we're not pushing a duplicate in that dict. Most likely the chances are minimal for that to happen but it doesn't harm to add it maybe.